doc/user/layouts/partials/create-source/envelope/debezium/details.html

```diff
@@ -1,12 +1,10 @@
 ### Debezium envelope details
 
-The Debezium envelope provides a "diff envelope", which describes the decoded
-records' old and new values; this is roughly equivalent to the notion of Change
-Data Capture, or CDC. Materialize can use the data in this diff envelope to
-process data as representing inserts, updates, or deletes.
+If you're using [Debezium] as your CDC tool, you can use either the default Debezium envelope or the [Open CDC](../../overview/open-cdc) envelope format designed by Materialize to prevent record duplication or missequencing.
 
-This envelope is called the Debezium envelope because it's been developed to
-explicitly work with [Debezium].
+The Debezium or Open CDC envelope provides a "diff envelope", which describes the decoded
```

Comment:
I should know this, but "diff envelope" may be jargon that we invented rather than a term of art. @umanwizard do you recall where we got this from? I suspect in any case that "diff envelope" is .. maybe confusing here. Idk.

Reply:
We can just make it "The Debezium envelope describes the decoded records' old and new values ..."